cms/djangoapps/contentstore/courseware_index.py

```diff
@@ -143,6 +145,16 @@ def index_item(item, skip_index=False):
             if not item_index_dictionary and not item.has_children:
                 return
 
+            if groups_usage_info:
+                item.content_groups = []
```

Comment:
I'm not understanding why these need to be an injected property of the item, wouldn't a variable `item_content_groups` suffice herein?